packages/nc-gui-v2/components/smartsheet/sidebar/menu/ApiSnippet.vue

```diff
@@ -0,0 +1,183 @@
+<script setup lang="ts">
+import HTTPSnippet from 'httpsnippet'
+import { useClipboard } from '@vueuse/core'
+import { useToast } from 'vue-toastification'
+import { ActiveViewInj, MetaInj } from '~/context'
+
+const props = defineProps<Props>()
+
+const emits = defineEmits(['update:modelValue'])
+
+const toast = useToast()
```

Comment:
can we use ant-design-vue `notification` instead of `toast`?
We were considering moving to the notifications anyway 😄